### PR TITLE
added a content-security-policy

### PR DESCRIPTION
**Pull request by avatus**
Closes: https://github.com/gravitational/teleport-private/issues/147
More info: https://github.com/gravitational/teleport-private/issues/96

Due to electron not being able to set HTTP request headers in production (because files are sent using the `file://` protocol), we can define a Content-Security-Policy in our "index.html" page.

This csp mostly matches the webui csp, with the exception of `font-src` where I've added `data:` to keep our icons working.

Note about `connect-src`: Adding in the urls `https://kcwm2is93l.execute-api.us-west-2.amazonaws.com/prod` and `https://usage.teleport.dev` are required for the feedback form to work on dev (insecure) and prod respectively

### CODE REVIEW

File: packages/teleterm/webpack.renderer.extend.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ function createHtmlPlugin() {
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
         <meta name="referrer" content="no-referrer" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta http-equiv="Content-Security-Policy" content="default-src 'self'; script-src 'self'; style-src 'self' 'unsafe-inline'; img-src 'self' data: blob:; object-src 'none'; font-src 'self' data:">
```
Comment (gzdunek) [inline packages/teleterm/webpack.renderer.extend.js:31]: We have to add two domains to `connect-src` directive for the share feedback form (it uses two URLs, please take a look at `useShareFeedback`).

Additionally, we can remove redundant `script-src` directive (it has the same value as `default-src`).

Comment (avatus) [reply packages/teleterm/webpack.renderer.extend.js]: thanks! done.

File: packages/teleterm/webpack.renderer.extend.js [hunk 2]
```diff
@@ -28,6 +28,7 @@ function createHtmlPlugin() {
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
         <meta name="referrer" content="no-referrer" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta http-equiv="Content-Security-Policy" content="default-src 'self'; connect-src 'self' https://kcwm2is93l.execute-api.us-west-2.amazonaws.com/prod; style-src 'self' 'unsafe-inline'; img-src 'self' data: blob:; object-src 'none'; font-src 'self' data:">
```
Comment (ravicious) [inline packages/teleterm/webpack.renderer.extend.js:31]: I think we're still missing the `https://usage.teleport.dev` URL here.

Comment (avatus) [reply packages/teleterm/webpack.renderer.extend.js]: whoops, removed during testing and forgot to throw back in. thank you!

**Review by gzdunek — APPROVED** [review]

**Review by ravicious — APPROVED** [review]

**Pull request merged**